### PR TITLE
Add tag issue support

### LINKED ISSUE #90: As a user, I can tag issues
... so that I can group similar issues

### PR DESCRIPTION
**Pull request by cnlinh**
## What this does
This PR
1. adds tag issue support

Closes #90 

## How to test
1. Launch SunRez.
2. Execute the `iadd r/10-100 d/Broken light t/2020/01/12 3:30pm s/pending c/furniture g/HIGH` command. You should see the issue added with the tag `HIGH`.

### CODE REVIEW

**Review by DrWala — COMMENTED** [review]
~Generally looks good, however, is there support for multiple tags? If there is, its currently not working.~

Update: It works now, probably some issue on my side.

**Review by DrWala — APPROVED** [review]
Tested and works properly for me.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -214,7 +214,7 @@ Example:
 
 Adds an issue to the housing management system.
 
-Format: `iadd r/ROOM_NO d/DESCRIPTION [t/TIMESTAMP] [s/STATUS] [c/CATEGORY]`
+Format: `iadd r/ROOM_NO d/DESCRIPTION [t/TIMESTAMP] [s/STATUS] [c/CATEGORY] [g/TAG]`
```
Comment (colintkn) [inline docs/UserGuide.md:217]: I didn't know how to use multiple tags at first. Can consider putting command example for both:
1. single tag 
`iadd r/10-100 d/Broken light c/Furniture g/ HIGH
2.  multiple tag
`iadd r/10-100 d/Broken light c/Furniture g/ HIGH g/ URGENT

**Review by colintkn — APPROVED** [review]
Other than having a clearer UserGuide, the feature works great. Thanks linh.

**Pull request merged**